### PR TITLE
Update Application Checks Documents

### PR DESCRIPTION
**Pull request by EyadLordXxXxX**
## Summary

This PR updates nextcord.ext.application_checks's documentations to be better/get fixed

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]
what is the purpose of the examples?

**Comment by Skelmis** [comment]
To, show an example usage? ext.commands show some example usages

**Comment by EyadLordXxXxX** [comment]
> what is the purpose of the examples?

It's supposed to show examples in the documentation, for example: ext.commands and ext.tasks have examples, ...

**Review by DenverCoder1 — APPROVED** [review]

File: nextcord/ext/application_checks/core.py [hunk 1]
```diff
@@ -360,6 +385,21 @@ def bot_has_any_role(*items: int) -> Callable[[T], T]:
...
+
+    Example
+    --------
+
+    .. code-block:: python3
+
+        @bot.slash_command()
+        @application_checks.bot_has_any_role('Moderators', 492212595072434186)
+        async def cool(interaction: Interaction):
+            await interaction.response.send_message('You are cool indeed')
```
Comment (DenverCoder1) [inline nextcord/ext/application_checks/core.py:402]: Change message?
```suggestion
            await interaction.response.send_message('I have a required role!')
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]

File: nextcord/ext/application_checks/core.py [hunk 2]
```diff
@@ -334,6 +344,21 @@ def bot_has_role(item: int) -> Callable[[T], T]:
...
+        The name or ID of the role to check.
+
+    Example
+    --------
+
+    .. code-block:: python3
+
+        @bot.slash_command()
+        @application_checks.bot_has_role(492212595072434186)
+        async def cool(interaction: Interaction):
```
Comment (ooliver1) [inline nextcord/ext/application_checks/core.py:360]:
```suggestion
        async def hasrole(interaction: Interaction):
```

**Review by ooliver1 — APPROVED** [review]

**Pull request merged**